README.md

```diff
@@ -115,7 +187,60 @@ The following intentions are required. In the examples, intentions are created i
 ## Outputs
 | Name | Description | Type |
 |------|-------------|------|
-| presto\_service\_name | Presto service name | string |
+| presto_service_name | Presto service name | string |
+
+## Secrets & credentials
+When using the `mode = "cluster`, you can set your secrets in two ways, either manually or upload secrets to Vault.
```

Comment:
We should potentially add an issue to make the same available in the standalone version

Reply:
Nvm, I'll do that in #61